client/my-sites/checkout/checkout-thank-you/jetpack-thank-you-card.jsx

```diff
@@ -26,7 +26,7 @@ import Card from 'components/card';
 import utils from 'lib/site/utils';
 import FormButtonsBar from 'components/forms/form-buttons-bar';
 import FormButton from 'components/forms/form-button';
-import HappyChatButton from 'components/happychat/button';
+import HappyChatButton from 'extensions/happychat/components/button';
```

Comment:
Same here, if we aren't able to remove the direct references to the extension, we should at least comment prominently that it needs to be resolved.